### ISSUE TITLE
Cherry pick methods from Lodash to reduce bundle size

### ISSUE THREAD

**Issue by ghost** [adamgibbons/ics#109]
It would be great if the lib cherry picks the lodash methods used to reduce bundle size.

Like:
```
// Cherry-pick methods for smaller browserify/rollup/webpack bundles.
const at = require('lodash/at');
const curryN = require('lodash/fp/curryN');

// OR
import at form 'lodash/at';
import curryN from 'lodash/fp/curryN';
```

instead of:

```
import _ from 'lodash';
```

**Comment by Akumzy** [comment]
solved [#111](https://github.com/adamgibbons/ics/pull/111l)

**Comment by adamgibbons** [comment]
Closed by #111
